packages/fxa-content-server/app/tests/spec/views/app.js

```diff
@@ -342,15 +408,66 @@ describe('views/app', function () {
       before(function () {
         createDeps();
 
-        sinon.spy(view, 'fatalError');
+        return view
+          .showView(ViewThatRenders, {
+            viewName: 'some-other-view',
+            surveyTargeter: getSurveyTargeter()
+          });
+      });
+
+      it('does not add the survey to the DOM', function () {
+        assert.equal($('.survey-wrapped').length, 0);
+      });
+    });
 
-        return view.showView(ViewThatErrors, {});
+    describe('with a survey, but no targeter', function () {
+      var ViewThatRenders = Backbone.View.extend({
+        render() {
+          this.$el.html('<div id="rendered-view"></div>');
+          return Promise.resolve(true);
+        },
+        titleFromView() {
+          return 'the title';
+        },
       });
 
-      it('writes the error to the DOM', function () {
-        assert.isTrue(view.fatalError.calledWith(renderError));
+      before(function () {
+        createDeps();
+        return view
+          .showView(ViewThatRenders, {
+            viewName: 'some-other-view'
+          });
       });
     });
+
+    it('does not add the survey to the DOM', function () {
+      assert.equal($('.survey-wrapped').length, 0);
+    });
+  });
+
```

Comment:
For these tests, we could spy on `_showSurvey` in the app view and check to see if it was called with the correct arguments.